packages/react/src/Dialog/Dialog.tsx

```diff
@@ -98,9 +98,9 @@ export interface DialogProps extends SxProp {
 
   /**
    * This method is invoked when a gesture to close the dialog is used (either
-   * an Escape key press or clicking the "X" in the top-right corner). The
+   * an Escape key press, clicking the backdrop, or clicking the "X" in the top-right corner). The
```

Comment:
should this reference to clicking the backdrop be removed since that information isn't actually communicated?